diff --git a/libraries/BluetoothSerial/src/BluetoothSerial.cpp b/libraries/BluetoothSerial/src/BluetoothSerial.cpp
index b20452c..4a0fa8f 100644
--- a/libraries/BluetoothSerial/src/BluetoothSerial.cpp
+++ b/libraries/BluetoothSerial/src/BluetoothSerial.cpp
@@ -510,7 +510,7 @@ static void esp_bt_gap_cb(esp_bt_gap_cb_event_t event, esp_bt_gap_cb_param_t *pa
                 confirm_request_callback(param->cfm_req.num_val);
             }
             else {
-                esp_bt_gap_ssp_confirm_reply(param->cfm_req.bda, true);
+//                esp_bt_gap_ssp_confirm_reply(param->cfm_req.bda, true);
             }
             break;
 
@@ -630,7 +630,7 @@ static bool _init_bt(const char *deviceName)
         log_i("Simple Secure Pairing");
         esp_bt_sp_param_t param_type = ESP_BT_SP_IOCAP_MODE;
         esp_bt_io_cap_t iocap = ESP_BT_IO_CAP_IO;
-        esp_bt_gap_set_security_param(param_type, &iocap, sizeof(uint8_t));
+        //esp_bt_gap_set_security_param(param_type, &iocap, sizeof(uint8_t));
     }
 
     // the default BTA_DM_COD_LOUDSPEAKER does not work with the macOS BT stack
@@ -793,7 +793,7 @@ void BluetoothSerial::onAuthComplete(AuthCompleteCb cb)
 
 void BluetoothSerial::confirmReply(boolean confirm)
 {
-    esp_bt_gap_ssp_confirm_reply(current_bd_addr, confirm);  
+//    esp_bt_gap_ssp_confirm_reply(current_bd_addr, confirm);  
 }
 
 
